buttons.css

```diff
@@ -0,0 +1,18 @@
+.button--border {
+    border: solid 1px var(--gray-2);
+    border-radius: 0.3em;
+    padding: 0.5em;
+}
+
+.button--without-border {
+    border: 0;
+}
+
+.button--border:hover {
+    color: var(---black);
+    border: solid 1px var(---black);
+}
+
+.button--without-border svg:hover {
+    fill: var(---black);
+}
```

Comment:
Let's refactor these styles to make more sense. We should have:

1. A `.btn` class that contains the base (common) styles
2. A `.btn--border` class that adds the border. This class should have just one rule, that adds the border. Nothing else.